lua/typescript-tools/config.lua

```diff
@@ -178,6 +180,13 @@ function M.load_settings(settings)
   if not M.code_lens_mode[settings.code_lens] then
     __store.code_lens = M.code_lens_mode.off
   end
+
+  if not settings.jsx_close_tag then
+    __store.jsx_close_tag = {
+      enable = true,
```

Comment:
I think is better to disable this by default because probably a lot  of people use some external plugin for this.
And in relation to this please add this option to readme

Reply:
Yep. I found that I have a local commit not push(f2bf21de90c5b317553a033a464ce42ca43b0bbb), I have disabled this feature in default.